README.md

```diff
@@ -256,20 +256,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
-                // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "irods_user_claim": "irods_username",
-
-                // The mapping of a user to the provided values. All values must
-                // be matched to map an OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "user_attribute_mapping": {
-                    "irods_username": {
-                        "sub": "123-abc-456-xyz",
-                        "email": "rods_user@example.org"
+                // User mapping plugins replace 'user_attribute_mapping' and
+                // 'irods_user_claim'. Addtional plugins may be developed to
+                // suit specific needs.
+                "user_mapping": {
+                    // Full path to plugin to load.
+                    // Currently we supply the following plugins:
+                    //   - local_file
+                    //   - user_claim
+                    "plugin_path": "/usr/lib/irods_http_api/plugins/user_mapping/libirods_http_plugin-local_file.so",
+
+                    // Configuration is plugin specific.
+                    // See your desired plugin's documentation for more details.
```

Comment:
Where should we document the plugin-specific configuration?

Reply:
i think it should live with the plugin.

the two that we are shipping... should probably be in this README directly.

and of course, if there is a third-party one... that's on them.

Reply:
In that case, unless there are other preferences, I'll put it in the OpenID Connect section.

Reply:
Done.